### PR TITLE
refactored installer.ts

### PR DESCRIPTION
**Pull request by lhein**
Signed-off-by: Lars Heinemann <lhein.smx@gmail.com>

### CODE REVIEW

File: src/IntegrationUtils.ts [hunk 1]
```diff
@@ -237,8 +237,8 @@ async function handleDefinedTask(context: vscode.Uri) {
 		await vscode.window.showInformationMessage('No Camel K Task applicable has been found. You can create one using "Tasks: Open User Tasks" or by creating a tasks.json file in .vscode folder.');
 	}
 
-	function isTheExactFile(file: string, context: vscode.Uri): boolean {
```
Comment (lhein) [inline src/IntegrationUtils.ts:240]: context was shadowing a global variable with that same name. That's why I renamed it.

**Comment by apupier** [comment]
tests are failing on CI

**Comment by lhein** [comment]
yep, and only there. locally all is fine.

**Comment by apupier** [comment]
> yep, and only there. locally all is fine.

all errors are related to a missing kamel.
what have you tried already to be closer to CI state?
removing kamel from the system path? using kamel 1.0.1?

**Comment by lhein** [comment]
build is green now

**Pull request merged**